File: programming files/even_odd_site.cc
#include <iostream>
#include <fstream>
using namespace std;

int main() {
    //creates file to write to
    ofstream file;
    //opens the file
    file.open("numbers.html");
    //adds a title and opens the body
    file << "<html>\n<head>\n<title>List of Numbers</title>\n</head>\n<body>\n";
    //creates a table
    file << "<table>\n<tr><th>Even Numbers</th><th>Odd Numbers</th></tr>\n";
    //loops through values 1-50
    for (int i = 1; i <= 50; i++) {
        if (i % 2 == 0) { //writes number to even column if its even
            file << "<tr><td>" << i << "</td><td></td></tr>\n";
        }
        else { //writes number to odd column if its odd
            file << "<tr><td></td><td>" << i << "</td></tr>\n";
        }
    }
    //ends the table and closes the body
    file << "</table>\n</body>\n</html>";
    //closes the file
    file.close();
    //creates input file
    ifstream input("numbers.html");
    //prints something from the file
    cout << input.rdbuf();
    //closes file
    input.close();
    //end of main, return code 0
    return 0;
}

File: programming files/even_odd_site.java


import java.io.*;

public class even_odd_site {
    public static void main(String[] args) throws IOException {
        // create file writer to write to number.html
        BufferedWriter file = new BufferedWriter(new FileWriter("numbers.html"));
        // write html code to file 
        file.write("<html>\n<head>\n<title>List of Numbers</title>\n</head>\n<body>\n");
        file.write("<table>\n<tr><th>Even Numbers</th><th>Odd Numbers</th></tr>\n");
        
        // write number to file
        for (int i = 1; i <= 50; i++) {
            // print even numbers
            if (i % 2 == 0) {
                file.write("<tr><td>" + i + "</td><td></td></tr>\n");
            }
            // print even numbers
            else {
                file.write("<tr><td></td><td>" + i + "</td></tr>\n");
            }
        }
        file.write("</table>\n</body>\n</html>");
        //close file writer
        file.close();
        // make new reader to print out line in FileReader
        BufferedReader reader = new BufferedReader(new FileReader("numbers.html"));
        String line;
        //reader whole file numbers.html
        while ((line = reader.readLine()) != null) {
            System.out.println(line);
        }
        //close reader
        reader.close();
    }
}
